modules/uncertainty/R/plots.R

```diff
@@ -8,26 +8,14 @@
 #-------------------------------------------------------------------------------
 
 ##--------------------------------------------------------------------------------------------------#
-##'  Plot results of variance decomposition
+##' Variance Decomposition Plots
 ##'
 ##' Plots variance decomposition tryptich
 ##' @name plot_variance_decomposition
-##' @title Variance Decomposition Plots
-##' @export plot_variance_decomposition
+##' @export
 ##' @author David LeBauer, Carl Davidson
-##' @param ... Output from any number of sensitivity analyses. Output must be of the form 
+##' @param plot.inputs Output from a sensitivity analysis. Output must be of the form
 ##'          given by sensitivity.results$variance.decomposition.output in model output
-##' @param all.plot.inputs Optional argument allowing output from sensitivity analyses to be specified in a list
-##' @param exclude vector of strings specifying parameters to omit from the variance decomposition graph
-##' @param convert.var function transforming variances to the value displayed in the graph
-##' @param var.label label to displayed over variance column
-##' @param order.plot.input Output from a sensitivity analysis that is to be used to order parameters.
-##'          Parameters are ordered by variance. Defaults to the first sensitivity analysis output given
-##' @param ticks.plot.input Output from a sensitivity analysis that is to be used.
-##'          Defaults to the first sensitivity analysis output given
-##' @param col Color of each sensitivity analysis. Equivalent to col parameter of the plot function.
-##' @param pch Shape of each sensitivity analysis. Equivalent to pch parameter of the plot function.
-##' @param main Plot title. Useful for multi-pft variance decompositions.
```

Comment:
@dlebauer this seems like ancient history (the function signature seems to have changed to remove these params in 2012) -- are we losing anything of value deleting these?